apps/docs/pages/migrating.mdx

```diff
@@ -0,0 +1,63 @@
+# Migrating
+
+## From v1 to v2
+
+Deroll v1 is based on Cartesi Rollups v1, while deroll v2 is based on Cartesi Rollups v2.
+
+Here are the changes you should be aware of when migrating from v1 to v2:
+
+### Input metadata
+
+The input metadata received in an advance handler has changed.
+
+```diff
++chain_id: number;
++app_contract: Address;
+msg_sender: Address;
+-epoch_index: number;
+input_index: number;
+block_number: number;
+-timestamp: number;
++block_timestamp: number;
++prev_randao: Hex;
+```
```

Comment:
To give some motivation, maybe explain why was `epoch_index` was removed, and why `chain_id`, `app_contract` and `prev_randao` have been added. Also, might be good to remark that `timestamp` was just renamed to `block_timestamp` and they still have the same semantics.

- Chain ID: can be useful to make the back-end network-agnostic.
- Application contract address: deprecates the old `DAppAddressRelay` contract.
- Previous RANDAO: can be used for randomness. I would link to [EIP-4399](https://eips.ethereum.org/EIPS/eip-4399#security-considerations) for security considerations and usage.